src/connectors/scenes/RequestConnector.js

```diff
@@ -64,6 +65,7 @@ const mapStateToProps = (state: State): RequestStateProps | RequestLoadingProps
     guiWallet,
     publicAddress: guiWallet.receiveAddress.publicAddress || '',
     legacyAddress: guiWallet.receiveAddress.legacyAddress || '',
+    segwitAddress: guiWallet.receiveAddress.segwitAddress || '',
```

Comment:
Could the prop have been `string | void`? I guess this works fine, but I generally dislike blank strings. Maybe a cleanup for another PR.

Reply:
lets leave for another PR, this was just doing what was there before, really not trying to do a refactor on this while trying to get it in.